src/shapes.ts

```diff
@@ -28,7 +28,12 @@ export interface ResourceFetchIdentity extends Identity {
     resourceFetch?: boolean;
 }
 
-export type EntityType = 'window' | 'iframe' | 'external connection' | 'unknown';
+export enum EntityType {
+   WINDOW = 'window',
+   IFRAME = 'iframe',
+   EXTERNAL = 'external connection',
+   UNKNOWN = 'unknown'
+}
```

Comment:
Do we need new entity type for browser view?

Reply:
Probably do